External/FEXCore/Source/Interface/Core/JIT/x86_64/JIT.cpp

```diff
@@ -406,6 +407,13 @@ X86JITCore::~X86JITCore() {
   FreeCodeBuffer(InitialCodeBuffer);
 }
 
+void X86JITCore::EmitDetectionString() {
+  const char JITString[] = "FEXJIT::X86JITCore::";
+  for (size_t i = 0; i < sizeof(JITString); ++i) {
```

Comment:
This can probably compress to:

```suggestion
  for (char c : JITString) {
```

Reply:
Resolved.